server/webapp/WEB-INF/vm/build_detail/_buildoutput.vm

```diff
@@ -14,30 +14,25 @@
  * limitations under the License.
  *************************GO-LICENSE-END***********************************#
 
-<div id="tab-content-of-console" $buildoutput_extra_attrs>
-    <div id="console-action-bar">
-        <div>
-            <a class='change-theme' href="javascript:void(0);">Change theme</a>
-            <a href="$req.getContextPath()/files/${presenter.consoleoutLocator}">Raw output</a>
+<div id="tab-content-of-console" class="ansi-color-toggle-$color_logs_feature_toggle_key" $buildoutput_extra_attrs>
+    #if($color_logs_feature_toggle_key)
+        <div class="console-action-bar">
+            <div>
+                <a class='change-theme' href="javascript:void(0);">Change theme</a>
+                <a href="$req.getContextPath()/files/${presenter.consoleoutLocator}">Raw output</a>
+            </div>
         </div>
-    </div>
+    #end
+    <pre class="buildoutput_pre #if(!$color_logs_feature_toggle_key) white-theme #end"></pre>
     #if($presenter.isCompleted())
-        <pre id="buildoutput_pre"></pre>
-        <div class="console_iframe_container"></div>
         <div class="widget" id="build-output-console-warnning" style="display: none;">No console output.</div>
-    #else
-        <pre id="buildoutput_pre"></pre>
     #end
 </div>
 
 <script type="text/javascript">
-    jQuery(function(){
-        jQuery('.change-theme').on('click', function(evt){
-            if (jQuery('#buildoutput_pre').length === 0){
-                return; // there's no theme to change
-            }
-            jQuery('#buildoutput_pre').toggleClass('white-theme');
-            jQuery('#console-action-bar').toggleClass('white-theme');
-        });
-    });
+    if(navigator && navigator.userAgent && navigator.userAgent.include('Firefox/2') && $('buildoutput_pre')){
+        (function(){
+            $('buildoutput_pre').up('div').setStyle({overflow: 'visible'});
```

Comment:
also, I think we can kill that section altogether, it's testing for really old versions of firefox that I don't think we need to care for anymore